examples/36_form/src/components/Form.tsx

```diff
@@ -22,7 +23,7 @@ export const Form = ({
   greet: (formData: FormData) => Promise<void>;
 }) => (
   <div style={{ border: '3px blue dashed', margin: '1em', padding: '1em' }}>
-    <p>{message}</p>
+    <p>{use(message)}</p>
```

Comment:
Some promises are rendered on client directly without `use`. I think React internally treats these two same and the bug were manifested in the same way. I updated to explicitly use `use`.

Reply:
It's a supported feature by React. This example is intentional. If it works, please revert it.

Reply:
I thought they still recommend explicitly unwrapping with `use` on client. Does React mention this on doc as a feature?

Reply:
Maybe now. But, let's keep the example unchanged unless it breaks.